libs/openFrameworks/gl/ofGLProgrammableRenderer.cpp

```diff
@@ -1315,12 +1321,46 @@ void ofGLProgrammableRenderer::bind(const ofFbo & fbo, bool setupPerspective){
 }
```

Comment:
oh sorry yeah, i didn't realize that is calling fbo.bind at the end since the diff view was hiding it :S